openpype/settings/entities/schemas/projects_schema/schemas/schema_openrv_publish.json

```diff
@@ -0,0 +1,50 @@
+{
+  "type": "dict",
+  "collapsible": true,
+  "key": "publish",
+  "label": "Publish plugins",
+  "children": [
+    {
+      "type": "dict",
+      "collapsible": true,
+      "checkbox_key": "enabled",
+      "key": "ValidateWorkfilePaths",
+      "label": "Validate Workfile Paths",
+      "is_group": true,
+      "children": [
+        {
+            "type": "boolean",
+            "key": "enabled",
+            "label": "Enabled"
+        },
+        {
+            "type": "boolean",
+            "key": "optional",
+            "label": "Optional"
+        },
+        {
+            "key": "node_types",
+            "label": "Node types",
+            "type": "list",
+            "object_type": "text"
+        },
+        {
+            "key": "prohibited_vars",
+            "label": "Prohibited variables",
+            "type": "list",
+            "object_type": "text"
+        }
+      ]
+    },
+    {
+      "type": "schema_template",
+      "name": "template_publish_plugin",
+      "template_data": [
+          {
+              "key": "ValidateContainers",
+              "label": "ValidateContainers"
+          }
+      ]
+    }
```

Comment:
Is this used for anything? Since the ValidateContainers plug-in doesn't appear in this PR?

Same goes for the `ValidateWorkfilePaths` above. Seems like a straight up copy from Houdini maybe and shouldn't actually be in here?

Reply:
yes, can be removed. there will be more leftovers needed for cleanup for sure...